src/components/Modal/__tests__/Modal.test.jsx

```diff
@@ -14,7 +14,10 @@ import { ModalContent } from '../ModalContent';
 import { ModalFooter } from '../ModalFooter';
 import { ModalHeader } from '../ModalHeader';
 
-describe('rendering', () => {
+// Test suites skipped due tu missing implementation of HTMLDialogElement in jsdom
```

Comment:
As it requires change of test environment, I'm preparing separate pull request with those changes.

Reply:
https://github.com/react-ui-org/react-ui/pull/545